### ISSUE TITLE
Improve drop-down menu size and provide feedback for mouseover

### ISSUE THREAD

**Issue by jonfroehlich** [ASSETS-Conference/assets2022#17]
From Slack.

> One quick finding: I don't think it's required by WCAG but I feel the navigation buttons can be improved. First, they are too small. The tapping area should be bigger than the text itself. This will enable people with motor disabilities to better use the site. Second, there is no visual indication when you hovering on them. While flashing animations are not recommended, having some visual indications will ease the cognitive loads for navigating the site

![image](https://user-images.githubusercontent.com/1621749/152427501-63ab6c75-f988-42e6-9f64-4c5c9f2a473c.png)
